src/HonzaBotner.Discord.Services/Commands/PollCommands.cs

```diff
@@ -0,0 +1,95 @@
+﻿using System.Linq;
+using System.Threading.Tasks;
+using DSharpPlus.CommandsNext;
+using DSharpPlus.CommandsNext.Attributes;
+using HonzaBotner.Discord.Extensions;
+using HonzaBotner.Discord.Services.Commands.Polls;
+
+namespace HonzaBotner.Discord.Services.Commands
+{
+    [Group("poll")]
+    [Description("Commands to create polls.")]
+    public class PollCommands : BaseCommandModule
+    {
+        private readonly string _pollErrorMessage =
```

Comment:
This should be const
```suggestion
        private const string PollErrorMessage =
```